test/test_manifest.json

```diff
@@ -68,7 +68,7 @@
     },
     {  "id": "issue3879",
        "file": "pdfs/issue3879.pdf",
-       "md5": "1cd1f1c3271515a14e65ff2980e14663",
+       "md5": "85393be14a304d55f516918a4df889e3",
```

Comment:
The new version of this file *does not* contain the same deficiency as the old one!